src/server/services/vergunningen.ts

```diff
@@ -300,6 +332,13 @@ export function createVergunningNotification(
   item: Vergunning,
   items: Vergunning[]
 ): MyNotification {
+  const notificationLinks: NotificationLinks = {
+    'Parkeerontheffingen Blauwe zone bedrijven':
+      'https://www.amsterdam.nl/veelgevraagd/?productid=%7B1153113D-FA40-4EB0-8132-84E99746D7B0%7D',
+    'Parkeerontheffingen Blauwe zone particulieren':
+      'https://www.amsterdam.nl/veelgevraagd/?productid=%7B1153113D-FA40-4EB0-8132-84E99746D7B0%7D', // Not yet available in RD
+    GPK: 'https://formulieren.amsterdam.nl/TripleForms/DirectRegelen/formulier/nl-NL/evAmsterdam/GehandicaptenParkeerKaartAanvraag.aspx/Inleiding',
```

Comment:
You can do: `[CaseType.GPK]: '...',`

Reply:
Also on lines 349 - 351

`item.caseType ==== CaseType.` basically everywhere where you find the values of the enum key, use the enum